samples/net/mqtt_publisher/boards/cc3220sf_launchxl.conf

```diff
@@ -0,0 +1,26 @@
+# Networking Config:
+CONFIG_NETWORKING=y
+CONFIG_NET_IPV4=y
+CONFIG_NET_IPV6=n
+CONFIG_NET_SOCKETS=y
+CONFIG_TEST_RANDOM_GENERATOR=y
+
+# Enable SimpleLink WiFi Driver and Socket Offload
+CONFIG_WIFI=y
+CONFIG_WIFI_SIMPLELINK=y
+CONFIG_NET_SOCKETS_OFFLOAD=y
+
+# Enable Secure Socket Offload
+CONFIG_TLS_CREDENTIAL_FILENAMES=y
+
+# Disable unneeded settings from the base prj.conf:
+CONFIG_DNS_RESOLVER=n
```

Comment:
Same here, we offload DNS call in all offloaded solutions I know (all 2).

Reply:
I see this being thrown in other examples, so I believe it is there for a reason. @GAnthony any thoughts on this on?

Reply:
Yes, I recall I had to explicitly turn off DNS_RESOLVER, otherwise Zephyr networking pulls in DNS stuff and starts trying to do DNS, which conflicts with SimpleLink.  
Ideally, Zephyr networking stack would need to know how to deal nicely with offloaded DNS, offloaded DHCP, offloaded TLS, etc....  But for now, this gets things to build/run correctly.